src/components/App/viewmodel.ts

```diff
@@ -165,6 +167,13 @@ export class AppViewModel {
       || this.video.current!.videoHeight == 0) {
       return;
     }
+    if (this.isMouseEnabled) {
```

Comment:
ditto.
Only hides the cursor when it's sending mouse_move event. i.e. `this.sharerEventChannel?.send` on line 180 being called